### PR TITLE
Refactoring of the Watchdog insight page

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Update the Watchdog insight page

### Motivation
There was very little details on the Watchdog insight page around what was displayed and how it was calculated for each product.
Leading each product to build their own page for this instead of centralising all the information into one. 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Hi @NBParis! I'm going to add the WIP label for now. Feel free to remove it when you're done. That will add it to our on-call queue

**Review by byassine93 — COMMENTED** [review]
A few comments

File: content/en/watchdog/insights.md [hunk 1]
```diff
@@ -19,79 +19,132 @@ further_reading:
...
+
+[1]: /real_user_monitoring/explorer/search/#facets
+[2]: /real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals
+[3]: /real_user_monitoring/browser/monitoring_page_performance/#monitoring-single-page-applications-spa
+{{% /tab %}}
+{{% tab "Serverless" %}}
+
+TODO
+
+For serverless infrastructures, Watchdog surfaces the following outliers:
```
Comment (byassine93) [inline content/en/watchdog/insights.md:147]: I still think those are not outliers per the definition at the beginning of the doc 
```Calculated on the product data matching the active query, it surfaces tags that either appear too frequently in some event types (e.g: errors) or drive some continuous metrics upwards (e.g: latency).```

File: content/en/watchdog/insights.md [hunk 2]
```diff
@@ -19,79 +19,132 @@ further_reading:
...
+**Latency Outlier**
+
+Latency outliers display fields such as [faceted tags or attributes][1] that are associated with performance bottlenecks that match the current search query. `key:value` pairs with worse performance than the baseline can provide hints into the performance bottlenecks among a subset of real users.
+
+Latency outliers are computed for [Core Web Vitals][2] such as First Contentful Paint, First Input Delay, Cumulative Layout Shift, and [Loading Time][3]. For more information, see [Monitoring Page Performance][2].
+
+In the **banner card** view, you can see:
+
+* The field name
+* The performance metric value containing the field and the baseline for the rest of the data
```
Comment (byassine93) [inline content/en/watchdog/insights.md:133]: + The distribution as well

File: content/en/watchdog/insights.md [hunk 3]
```diff
@@ -19,79 +19,132 @@ further_reading:
...
+{{< img src="watchdog/log_explorer_watchdog_insights.png" alt="The Watchdog Insights carousel on the Logs Explorer, showing three anomalies: new error logs in the web-store service, a spike in error logs in the product-recommendation service, and another spike in error logs in the product-recommendation service" >}}
+
+Click **View all** to expand the panel. A side panel opens from the right, containing a vertical list of Watchdog Insights. Each entry shows a detailed view, with more information than the summary card.
 
-### Filter on Insight
+Every insight comes with embedded interactions and a side panel with troubleshooting information. The insight interactions and side panel vary based on the Watchdog Insight type.
+
+**Filter on Insight query**:
 
 To refine your current view to match a Watchdog Insight, hover over the top right corner of an insight summary card. Two icons appear. Click on the inverted triangle icon with the tooltip **Filter on Insight**. The page refreshes to show a list of entries corresponding to the insight.
```
[the remark this answers is not in the document]
Comment (NBParis) [reply content/en/watchdog/insights.md:58]: As discussed, will rephrase to `outliers` instead of insights

**Comment by hestonhoffman** [comment]
I created [DOCS-6077](https://datadoghq.atlassian.net/browse/DOCS-6077) for someone on the docs team to take a look.

**Review by cswatt — APPROVED** [review]

**Pull request merged**